src/_actions/__tests__/AppConfigActions-test.js

```diff
@@ -0,0 +1,14 @@
+import { UPDATE_APP_CONFIG } from '../../_constants/ActionTypes';
+import {updateAppConfig} from '../AppConfigActions';
+import expect from 'expect';
+
+describe('AppConfigActions',()=>{
+  it('should be equal to the expectedAction',()=>{
+      const expectedAction = {
+        type: UPDATE_APP_CONFIG,
+        field: 'balance',
+        value: 3,
+      };
+      expect(updateAppConfig('balance',3)).toEqual(expectedAction);
```

Comment:
@borisyankov @nuruddeensalihu do you think we can skip these test, seems not very valuable ~

I think testing on action creators is more worth it, as there's logic inside

Thoughts?